src/MOI_wrapper.jl

```diff
@@ -291,6 +309,21 @@ function MOI.get(
     return MOI.get(model.context.model, attr, ci)
 end
 
+# See `constraints/constraints.jl`
+# `LessThan` constraints are reformulated as `rhs - lhs` unlike MOI while
+# `GreaterThan` constraints are reformulated as `lhs - rhs` like in MOI
+_flip_dual(x, ::Type{S}) where {S<:MOI.LessThan} = -x
+_flip_dual(x, ::Type{S}) where {S<:MOI.AbstractScalarSet} = x
+
+function MOI.get(
+    model::Optimizer,
+    attr::MOI.AbstractConstraintAttribute,
```

Comment:
Shouldn't this be `MOI.ConstraintDual` or something? Why are we negating every attribute?

Reply:
Yes, and ConstraintPrimal